src/whiteboard/manager.cpp

```diff
@@ -869,6 +873,9 @@ bool manager::save_recruit(const std::string& name, int side_num, const map_loca
 			created_planned_recruit = true;
 
 			print_help_once();
+#else
+			created_planned_recruit = false;
```

Comment:
You need to add a couple of UNUSED(...) here to deal with the unused parameters. This is unfortunately the reason travis failed, and is a merge blocker until fixed.